ui/components/ui/chip/chip.scss

```diff
@@ -42,6 +42,8 @@
     border: none;
     background: transparent;
     text-align: center;
+    width: 100%;
+    font-size: 16px;
```

Comment:
use the typography components or scss variables instead of literal values.

Reply:
fixed